src/olympia/discovery/admin.py

```diff
@@ -38,5 +42,26 @@ def formfield_for_foreignkey(self, db_field, request=None, **kwargs):
         return super(DiscoveryItemAdmin, self).formfield_for_foreignkey(
             db_field, request, **kwargs)
 
+    def build_preview(self, obj):
+        return format_html(
+            u'<br/><h2 class="heading">{}</h2>'
+            u'<div class="editorial-description">{}</div>',
+            mark_safe(obj.heading),
+            mark_safe(obj.description))
+
+    def previews(self, obj):
+        translations = []
```

Comment:
I'll update that with the key locales we're interested in: `('en-US', 'en-GB', 'de', 'fr', 'ru', 'pt-BR', 'es-ES', 'pl', 'it', 'ar', 'ja')`